README.md

```diff
@@ -87,6 +87,12 @@ module.exports.webpack = function(config) {
 };
 ```
 
+You can use this functionality to inject external styles or scripts if your
+components need e.g. Bootstrap to work.
+```js
+config.entry.push('../../../injectStyleTag.js');
```

Comment:
> Also, since you gave Bootstrap as an example, it might be useful to have the bootstrap file in the example as well. (unless you need many extra files and it looks ugly, I guess...)

I suppose most use cases would include external file from CDN, in which case you need a custom JS file to inject actual script/style tags.

> This example is a bit confusing, why would be the extra file be outside the project (../../..), since the config file is located in the root of your project, right?

Using `./injectStyleTag.js` would look for the file in cosmos project inside `node_modules`. In this case the file `injectStyleTag.js` is actually in the root of my project. I agree it looks a bit odd, but I'm not sure how to get around it.